tests/unit/test__datastore_api.py

```diff
@@ -270,6 +270,30 @@ class SomeKind(model.Model):
 
         assert global_cache.get([cache_key]) == [cache_value]
 
+    @staticmethod
+    @mock.patch("google.cloud.ndb._datastore_api._LookupBatch")
+    def test_cache_miss_lock_not_acquired(_LookupBatch, global_cache):
```

Comment:
I'm confused about your confusion. ;-)

The intent is just what the method name says. We're testing the code path if there's a cache miss, but the attempt to acquire a lock fails (because someone set the key in the meantime). So, to your question, yes.

I'm a prisoner of my own foreknowledge, though. Is "lock_not_acquired" not clear about the intent? What would you call it?

Reply:
To me, the name says "Test that we don't acquire a lock on a cache miss", which is clearly wrong..  Maybe "test_lock_not_acquired_if_write_after_cache_miss".

Reply:
fa67d2c